client/driver/rkt.go

```diff
@@ -160,6 +156,20 @@ func (d *RktDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle, e
 	// Append the run command.
 	cmdArgs = append(cmdArgs, "run", "--mds-register=false", img)
 
+	// Mount allc and task dirs
+	binds, err := getBindDirs(ctx.AllocDir, task)
+	if err != nil {
+		return nil, err
+	}
+	for _, bind := range binds {
+		s := strings.Split(bind, ":")
```

Comment:
You aren't really using the output of the getBindDirs directly, you are parsing its output. So rather create a specialized method that returns what you are doing below. There doesn't have to be shared code between the rkt/docker driver as they are outputting different formats.